carbonmark-api/src/graphql/digitalCarbon.gql

```diff
@@ -18,12 +18,16 @@ query getDigitalCarbonProjectsCountries {
   }
 }
 
-query getProjectCredits($projectID: String!, $vintage: Int) {
+query getProjectCredits(
+  $projectID: String!
+  $vintage: Int
+  $minSupply: BigInt
+) {
   carbonProjects(where: { projectID: $projectID }) {
     ...DigitalCarbonProjectFragment
     carbonCredits(where: { vintage: $vintage }) {
       ...CarbonCreditFragment
-      poolBalances {
+      poolBalances(where: { balance_gt: $minSupply }) {
```

Comment:
@biwano `min` implies `gte`, not `gt`

Reply:
Thanks! But I noticed an issue with the updatedAt value. I cannot not filter directly in the query. So these got removed.